keyboards/contra/keymaps/erovia/keymap.c

```diff
@@ -0,0 +1,176 @@
+/* Copyright 2015-2017 Jack Humbert
```

Comment:
You don't need to put Jack's name on your keymap.c file. You can put your name here since it's your keymap.